lib/senkyoshi/models/qti.rb

```diff
@@ -10,15 +10,16 @@
 }.freeze
 
 module Senkyoshi
-  class QTI < Resource
-    def self.from(data, pre_data)
+  class QTI < FileResource
+    def self.from(data, pre_data, _ = nil)
```

Comment:
I don't have a problem with doing `_ = nil` but I prefer to give it a name rather than just `_`. But that is just my ocd. :]